src/module-elasticsuite-indices/Block/Adminhtml/Analysis/Analyzer.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteIndices
+ * @author    Vadym HONCHARUK <vahonc@smile.fr>
+ * @copyright 2022 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+namespace Smile\ElasticsuiteIndices\Block\Adminhtml\Analysis;
+
+use Exception;
+use Magento\Backend\Block\Template;
+use Smile\ElasticsuiteIndices\Model\IndexStatsProvider;
+use Smile\ElasticsuiteIndices\Model\ResourceModel\IndexSettings\CollectionFactory as IndexSettingsFactory;
+
+/**
+ * Adminhtml Analysis by Analyzer Block.
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteIndices
+ * @author   Vadym HONCHARUK <vahonc@smile.fr>
+ */
+class Analyzer extends Template
+{
+    public const GHOST_STATUS = 'ghost';
```

Comment:
I think those constants exists elsewhere in the same module, no ?

Reply:
Yes, you're right. I fixed this too.